xml/storage_multipath.xml

```diff
@@ -4617,32 +4617,91 @@ Timed out waiting for device dev-disk-by\x2duuid-c4a...cfef77d.device.
 Welcome to emergency shell
 Give root password for maintenance
 (or press Control-D to continue):</screen>
-   <para>
-    This issue occurs in the following situations:
-   </para>
-   <itemizedlist>
-    <listitem>
-     <para>
-      A non-multipath root file system is not blacklisted when multipath is
-      enabled. See
-      <xref linkend="pro-multipath-trouble-root-blacklist"
-       xrefstyle="HeadingOnPage"/>.
-     </para>
-    </listitem>
-    <listitem>
-     <para>
-      On a system with a multipath root file system, multipath has been
-      enabled/disabled without rebuilding the <filename>initrd</filename>. See
-      <xref linkend="pro-multipath-trouble-root-initrd"/>.
-     </para>
-    </listitem>
-    <listitem>
+    <para>
+     At this stage, you are working in a temporary <command>dracut</command>
+     emergency shell from the initrd environment. To make the configuration
+     changes described below persistent, you need to perform them in the
+     in the environment of the installed system.
+    </para>
+    <important>
      <para>
-      Network-storage drivers are missing in the <filename>initrd</filename> on
-      a system with network-attached storage and multipath enabled.
+      As a general rule, rebuild <filename>initrd</filename> whenever
```

Comment:
Why mention this here, in the middle of explaining how to chroot?

Reply:
It's absolutely true, but should rather be mentioned where we explain how to make `multipath.conf` changes. (I suppose there's already something like that in the docs).